spec/unit/speak_slack_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../app/speak_slack'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe SpeakSlack do
+    describe 'SpeakSlack says' do
+      context 'when passed an empty array' do
+        it 'food trucks cannot be found' do
+          expect(SpeakSlack.trucks).to be(MSG_NO_FOOD_TRUCKS)
```

Comment:
This expectation is more specific (and therefore more brittle) than needed.  There is no user-observable difference between the method returning the exact `MSG_NO_FOOD_TRUCKS` string instance (which `be(MSG_NO_FOOD_TRUCKS)` requires to pass) and a different string instance containing the same contents -- but if your code returned a different (but equivalent) string instance this spec would fail.  I'd recommend using `eq` instead.